packages/now-node/tsconfig.json

```diff
@@ -6,7 +6,7 @@
     "target": "esnext",
     "module": "commonjs",
     "outDir": "dist",
-    "sourceMap": false,
+    "sourceMap": true,
```

Comment:
Is this intentional?

Reply:
I'm trying to decide… on the fence. Basically we could enable source maps for the spawned dev server process, but in the case of this option, only the `src/dev-server.ts` file would get the benefit. So it's an improvement in that case, but perhaps the package size trade-off isn't worth it? 🤔

Reply:
> perhaps the package size trade-off isn't worth it

That was my first thought. If the dev server isn't minified than it probably isn't needed but maybe you can show a before/after where the spawn dev process throws so we can compare?

Reply:
Reverted the change.

Reply:
It doesn't seem reverted? I still see the change 🤔

Reply:
Sorry, there was a push conflict. Fixed.